### PR TITLE
:green_heart: (action) fix wrong file path

### PR DESCRIPTION
**Pull request by david30907d**

### CODE REVIEW

File: .github/workflows/osm_parquetizer.yml [hunk 1]
```diff
@@ -33,12 +33,11 @@ jobs:
           ${{ runner.os }}-buildx-
 
     # And make it available for the builds
-    - name: Build neo4j and push
+    - name: Build Osm Parquetizer and push
```
Comment (david30907d) [inline .github/workflows/osm_parquetizer.yml:36]: forgot to rename

File: dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
 FROM maven:3.8.4-jdk-11-slim
-COPY ./pipelines/osm-poi/osm-parquetizer /opt/app
+COPY . /opt/app
```
Comment (david30907d) [inline dockerfile:2]: it's @mattigrthr 's fix actually

**Comment by david30907d** [comment]
passed CI 
https://github.com/david30907d/osm-parquetizer/actions/runs/1601750741

**Review by mattigrthr — APPROVED** [review]
Thanks for the quick fix!

**Pull request merged**